### PR TITLE
[build-publish] support debian-11-worker-arm64

### PR DESCRIPTION
**Pull request by dntczdx**

### CODE REVIEW

File: daisy_workflows/build-publish/debian/debian_11_worker_arm64.wf.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "Name": "build-debian-11-worker-arm64",
+  "Project": "gce-image-builder",
+  "Zone": "us-central1-a",
+  "GCSPath": "gs://gce-image-build-bucket/daisy/${USERNAME}",
+  "Vars": {
+    "build_date": {
+      "Value": "${DATE}w",
```
Comment (zmarano) [inline daisy_workflows/build-publish/debian/debian_11_worker_arm64.wf.json:8]: extra w?

File: daisy_workflows/build-publish/debian/debian_11_worker_arm64.wf.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      "Description": "Build datestamp used to version the image."
+    },
+    "google_cloud_repo": {
```
Comment (zmarano) [inline daisy_workflows/build-publish/debian/debian_11_worker_arm64.wf.json:11]: I think we can remove this?

File: daisy_workflows/build-publish/debian/debian_11_worker_arm64.wf.json (hunk 1 above)
Comment (dntczdx) [reply daisy_workflows/build-publish/debian/debian_11_worker_arm64.wf.json:8]: it exists in debian-10-worker. I thought it's by purpose. Let me remove both then.

File: daisy_workflows/build-publish/debian/debian_11_worker_arm64.wf.json (hunk 2 above)
Comment (dntczdx) [reply daisy_workflows/build-publish/debian/debian_11_worker_arm64.wf.json:11]: done

File: daisy_workflows/build-publish/debian/debian_11_worker_arm64.wf.json (hunk 1 above)
Comment (zmarano) [reply daisy_workflows/build-publish/debian/debian_11_worker_arm64.wf.json]: Oooh it does. It is to differentiate the worker image from the normal one. Ok yeah lets leave it.

**Pull request merged**